components/script/dom/bluetoothdevice.rs

```diff
@@ -59,13 +68,53 @@ impl BluetoothDeviceMethods for BluetoothDevice {
         self.name.clone()
     }
 
-    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothdevice-addata
-    fn AdData(&self) -> Root<BluetoothAdvertisingData> {
-        self.adData.get()
-    }
-
     // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothdevice-gatt
     fn Gatt(&self) -> Root<BluetoothRemoteGATTServer> {
         self.gatt.or_init(|| BluetoothRemoteGATTServer::new(self.global().r(), self))
     }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothdevice-watchadvertisements
+    fn WatchAdvertisements(&self) -> ErrorResult {
+        if self.watchingAdvertisements.get() {
+            return Err(Type(WRONG_FUNC_CALL.to_owned()));
+        }
+        let (sender, receiver) = ipc::channel().unwrap();
+        self.get_bluetooth_thread().send(
+            BluetoothMethodMsg::WatchAdvertisements(self.id.to_string(), sender)).unwrap();
+        let result = receiver.recv().unwrap();
```

Comment:
There isn't any empty line in any of the functions in the DOM side. Wouldn't it be better to do this with every file later in a style fix, or something?